2-Draft-Accepted/RFCnnnn-PowerShell-as-Login-Shell.md

```diff
@@ -70,10 +62,17 @@ The downsides to this approach is additional code to maintain,
 but more importantly not getting a complete login shell environment as things
 like ulimit and umask would not be inherited into pwsh.
 
-### pwsh to start `/bin/sh -l -c "exec pwsh"`
+### `pwsh-login` script
+
+This proposal would be to include a Bourne shell script called `pwsh-login`
+that would contain:
+
+```sh
+#!/bin/sh -l
+exec /usr/local/bin/pwsh "$@"
+```
 
-This proposal would have pwsh when given the `-l` switch to start Bourne shell
-as a login shell and start pwsh within that process.
+The location of `sh` would depend on the specific operating system.
 
-This would result in a complete login shell environment, however, would
-incur the performance penalty of starting pwsh twice.
+The perf impact of this compared to starting pwsh as a non-login shell is
+approximately 4%.
```

Comment:
In line 46 - 6%.

Reply:
The 6% is starting `pwsh` from `pwsh`.  This 4% is a perf impact from the script in line 70.  So comparing the two implementations, you get a small improvement in perf from a script vs code.